core/src/test/java/org/apache/accumulo/core/iterators/system/SourceSwitchingIteratorTest.java

```diff
@@ -286,10 +286,8 @@ public void testSetInterrupt() throws Exception {
 
     flag.set(true);
 
-    try {
-      ssi.seek(new Range("r1"), new ArrayList<>(), false);
-      fail("expected to see IterationInterruptedException");
-    } catch (IterationInterruptedException iie) {}
+    assertThrows(IterationInterruptedException.class,
+        () -> ssi.seek(new Range("r1"), new ArrayList<>(), false));
```

Comment:
Presumably, these are failing on the `seek` method, and not on `new Range`. But the code isn't clear. It could be written like:

```suggestion
    var range = new Range("r1");
    var newList = new ArrayList<?>();
    assertThrows(IterationInterruptedException.class, () -> ssi.seek(range, newList, false));
```

In general, it's best to assert the throws on the narrow-est method possible. The old code is sub-par for this reason as well, so it's not a problem you introduced. But, it would be a good opportunity to clean these up.

Reply:
Addressed in 2c61494